ds3-cli-helpers/src/test/java/com/spectralogic/ds3cli/helpers/ABMTestHelper.java

```diff
@@ -101,7 +89,7 @@ public static void deleteDataPolicy(
         try {
             final DeleteDataPolicySpectraS3Response deleteDataPolicy = client
                     .deleteDataPolicySpectraS3(new DeleteDataPolicySpectraS3Request(dataPolicyName));
-            assertThat(deleteDataPolicy.getStatusCode(), is(204));
+            Assert.assertThat(deleteDataPolicy.getStatusCode(), CoreMatchers.is(204));
```

Comment:
I would change the import for `CoreMatchers` to be a static import, so that way you can omit `CoreMatchers` from `CoreMatchers.is`